airbyte-integrations/connector-templates/source-configuration-based/metadata.yaml.hbs

```diff
@@ -7,6 +7,10 @@ data:
       enabled: false
     cloud:
       enabled: false
+  connectorBuildOptions:
+    # Please update to the latest version of the connector base image.
+    # https://hub.docker.com/r/airbyte/python-connector-base
+    baseImage: docker.io/airbyte/python-connector-base:1.0.0@sha256:dd17e347fbda94f7c3abff539be298a65af2d7fc27a307d89297df1081a45c27
```

Comment:
My vision here is always to use the `latest` tag instead of the pinned version unless it's necessary to pin it as a temporary workaround or something else. WDYT?

Reply:
We don't want to use `latest` here as it mean the base image used to build the connector can change anytime, without connector code change. We want to pin the base image to a specific tag to make sure two builds  on the same commit use the same base image. We also want any change to the `baseImage` metadata to be tied to a connector version bump.